specification.markdown

```diff
@@ -653,16 +626,10 @@ description: The minimum `DocType` version an `EBML Reader` has to support to re
 
 name: CRC-32
 
-level: 1
-
-global: true
+path: `/*/CRC-32(0,1)`
```

Comment:
This is different. In the earlier definition we had said `EBML Elements defined as global and recursive MAY occur at a level greater than or equal to the defined level.` so CRC-32 could occur within any Master Element so that CRC-32 had a position of Level 1 or greater.

This `path` implies that CRC-32 can not be at Level 2 but only at Level 1.

Perhaps here we need a `/*//CRC-32(0,1)` where `//` means at any place.